c-runtime/Makefile

```diff
@@ -89,11 +101,40 @@ doc:
 lint:
 	clang-tidy --checks=*,-bugprone-reserved-identifier,-cert-dcl37-c,-cert-dcl51-cpp,-altera-struct-pack-align $(SRC)
 
+#https://github.com/catchorg/Catch2/blob/v2.x/include/catch.hpp
+.PHONY: curl_catch
+curl_catch:
+	curl -o $(TESTDIR)/include/catch.hpp https://raw.githubusercontent.com/catchorg/Catch2/v2.x/single_include/catch2/catch.hpp
```

Comment:
@0sak Das sieht irgendwie so harmlos aus. Aber wenn Du mal in die Datei reinschaust, die hier referenziert wird, dann steht da **"Catch _v2.13.10_ Generated: 2022-10-16 11:01:23.452308"** drin. 

Das Moped ist uralt!

Die aktuelle Version wäre **v3.6.0**, etwa via https://github.com/catchorg/Catch2/releases/tag/v3.6.0. Wobei mir unklar ist, dort gibt es eine .cpp-Variante und eine .hpp-Variante - was genau ist der Unterschied?

Reply:
OK, lt. https://github.com/catchorg/Catch2/blob/devel/docs/migrate-v2-to-v3.md#how-to-migrate-projects-from-v2-to-v3 willst du beide Dateien haben (Vorgehen (1)). 

D.h. beide Dateien runterladen und in den Source-Tree werfen, und in den Test-Dateien ein `#include "catch_amalgamated.hpp"`.